src/validator/CreditCardChecking.java

```diff
@@ -0,0 +1,5 @@
+package validator;
+
+public interface CreditCardChecking {
+	public boolean checkValidity(CreditCardVendor ccv, String number);
+}
```

Comment:
In general, this interface is implemented only by CreditCardValidator. In my opinion, you can get rid of it, because we don't expect a second implementation.